qiita_client/__init__.py

```diff
@@ -12,6 +12,24 @@
 from .plugin import (QiitaCommand, QiitaPlugin, QiitaTypePlugin,
                      QiitaArtifactType)
 
+import logging
+
+handler = logging.StreamHandler()
+fmt_str = '%(asctime)s %(name)-12s %(levelname)-8s %(message)s'
+handler.setFormatter(logging.Formatter(fmt_str))
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+
+# logging level can be set to one of five levels:
```

Comment:
It looks like we can't control the logging level from outside the code, is this possible to add? I can imagine that this could be controlled via an environment variable, which if not set will be set to CRITICAL ?

Reply:
I haven't tested it yet, but I believe we can change it directly by going through the qclient object: qclient.logger.setLevel(logging.DEBUG). My thinking was to add functions to qiita_client util to wrap this, so we wouldn't need to import logging.DEBUG, etc.

Reply:
We could also have it pull it from an env variable, that would work well for travis builds, etc.

Reply:
Right, env var was thinking in travis and our own systems. Thanks!